docs/products/mysql/howto/connect-with-java.rst

```diff
@@ -0,0 +1,55 @@
+Connect to MySQL with Java
+============================
+
+This example connects your Java application to a MySQL service.
+
+Variables
+'''''''''
+
+These are the placeholders you will need to replace in the code sample:
+
+==================      =============================================================
```

Comment:
would be possible to change the table format to this: https://sublime-and-sphinx-guide.readthedocs.io/en/latest/tables.html?